.github/workflows/test.yml

```diff
@@ -135,10 +135,11 @@ jobs:
         id: build
         run: |
           foreach ($target in ($env:RUST_TARGETS).Split(",")) {
-            echo "Building for target: $target"
+            echo "::group::Building for target: $target"
             cargo build -vv --features use-bindgen $(if ($target -ne 'default') {"--target=$target"} )
+            echo "::endgroup::"
```

Comment:
Doesn't the endgroup statement have to come after the `if` statement checking for errors?

Reply:
I put "groups" around `cargo` commands so log under spoiler contains only build/test related output.
If `if` condition is met and error is thrown, it will be visible separtely.

Reply:
Yes, but see: https://github.com/extendr/extendr/pull/130#discussion_r558802290
Does this not apply here?

Reply:
Oh, I see, that is indeed very stupid of me. I will fix it with the next commit.